client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/toggle-activate-monitoring/index.tsx

```diff
@@ -144,11 +144,12 @@ export default function ToggleActivateMonitoring( {
 	};
 
 	const toggleContent = (
+		// For Atomic sites which do not support monitoring, we show the toggle as disabled while in active state.
 		<ToggleControl
 			onChange={ handleToggleActivateMonitoring }
-			checked={ isChecked }
-			disabled={ isLoading || siteError }
-			label={ isChecked && currentSettings() }
+			checked={ site.is_atomic || isChecked }
```

Comment:
What if the user has disabled the monitor?

Reply:
For disabled monitoring in non-Atomic sites, we still need the component enabled to allow them to switch the toggle. As far as I know, Atomic sites cannot disable the monitor in JC.

Reply:
One can disable it from WPCOM > Settings > Security.

<img width="762" alt="Screenshot 2023-10-16 at 3 09 07 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/935e2a57-95ba-47de-874d-91ddb1f964f8">

Reply:
I think we can still use the `isChecked ` for checked prop 🙂

Reply:
Good catch. Didn't know we could do this on atomic sites. I'll make the changes to account for this 👍